app/controllers/nonsipp/ViewOnlyTaskListController.scala

```diff
@@ -611,12 +659,24 @@ object ViewOnlyTaskListController {
   ): TaskListSectionViewModel = {
     val prefix = "nonsipp.tasklist.otherassets"
 
-    val quotedSharesStatusAndLink: TaskListStatus = getCompletedOrUpdatedTaskListStatus(
+    val quotedSharesCompletedOrUpdated: TaskListStatus = getCompletedOrUpdatedTaskListStatus(
       currentUA,
       previousUA,
       Paths.shares \ "totalValueQuotedShares"
     )
 
+    val quotedSharesTaskListStatus = quotedSharesCompletedOrUpdated match {
+      case Completed =>
+        currentUA.get(TotalValueQuotedSharesPage(srn)) match {
+          case Some(amount) if !amount.isZero => Reported
+          // The condition above is necessary because there is no boolean field in ETMP where we can store the answer to
+          // the first question page, so a value of 0.00 stored in ETMP indicates that no Quoted Shares were reported.
+          case _ => Reported("None")
```

Comment:
how is Reported("None") interpreted? Does it display none or is it consumed by some logic to output something else?

Reply:
On the Task List page the status will display as "None reported", is that what you mean in terms of how things are interpreted? Or are you asking more about how our code handles the `Reported` status internally, rather than how it's rendered?

Reply:
Yeah it was more about how the code handles it - if it's comparing the string value and checking if it's the value "None", it might make more sense to have that value at the object level e.g. `Reported.None` or something like that - this is minor though but in my experience arbitrary string comparison could lead to issues with miss-spelling, case sensitivity, etc but this is just a nitpick